### PR TITLE
fix/feat: Rewriten User Management page in reactJS

### PR DESCRIPTION
**Pull request by adityadeshlahre**
# fixes (#1042)

# Pull Requests Requirements

- [x] The PR title includes a brief description of the work done, including the Issue number if applicable.
- [x] The PR includes a video showing the changes for the work done.
- [x] The PR title follows conventional commit label standards.
- [x] The changes confirm to the OpenElis Global x3 Styleguide and design documentation.
- [x] The changes include tests or are validated by existing tests.
- [x] I have read and agree to the Contributing Guidelines of this project.

## Summary
- MVC to Rest Controller
- JSP to React JS page

### Main Page
image

### Add/Edit Page
image
image

### WorkFlow Video Soon

[Add relevant screenshots here if applicable]

## Related Issue

[Add a link to the related issue or mention it here if applicable]

## NOTE : Few Components need to be added
- some logics are not working
- edge cases are not test yet
- selectable component left

## 60 % ready 

[Add any additional information or notes here]

If any changes are required please let me know
Thank You :slightly_smiling_face:

### CODE REVIEW

File: frontend/src/components/admin/Admin.js [hunk 1]
```diff
@@ -57,6 +59,9 @@ function Admin() {
           <SideNavLink renderIcon={ChartBubble} href="#program">
             <FormattedMessage id="sidenav.label.admin.program" />
           </SideNavLink>
+          <SideNavLink renderIcon={User} href="#userManagement">
+            <FormattedMessage id="unifiedSystemUser.browser.title" />
```
Comment (sherrif10) [inline frontend/src/components/admin/Admin.js:63]: Hello @adityadeshlahre , some merge conflicts to fix

Comment (adityadeshlahre) [reply frontend/src/components/admin/Admin.js:63]: i will fix it once PR is 100% ready :slightly_smiling_face:

File: src/main/java/org/openelisglobal/systemuser/validator/UnifiedSystemUserFormValidator.java [hunk 2]
```diff
@@ -20,7 +20,7 @@ public boolean supports(Class<?> clazz) {
     public void validate(Object target, Errors errors) {
         UnifiedSystemUserForm form = (UnifiedSystemUserForm) target;
 
-        if (!form.getUserPassword().matches(UnifiedSystemUserController.DEFAULT_OBFUSCATED_CHARACTER + "+")) {
+        if (!form.getUserPassword().matches(UnifiedSystemUserRestController.DEFAULT_OBFUSCATED_CHARACTER + "+")) {
```
Comment (mozzy11) [inline src/main/java/org/openelisglobal/systemuser/validator/UnifiedSystemUserFormValidator.java:23]: any reason why you changesd this??

Comment (adityadeshlahre) [reply src/main/java/org/openelisglobal/systemuser/validator/UnifiedSystemUserFormValidator.java:23]: just to maintain symmetry

**Comment by adityadeshlahre** [comment]
Thanks @mozzy11 for solving the bug!

This PR is completed let me know if any changes are required :slightly_smiling_face:
